docs/fusion.md

```diff
@@ -9,317 +9,33 @@
 
-This configuration requires the the use of Nextflow local execution and Docker (or similar container engine) for the
-execution of your pipeline tasks.
+To enable Fusion in your Nextflow pipeline, add the following snippet to your `nextflow.config` file:
 
-For the same of this example, runs a local instance of Minio using this command:
-
-```
-docker run -p 9000:9000 \
-    --rm -d -p 9001:9001 \
-    -e "MINIO_ROOT_USER=admin" \
-    -e "MINIO_ROOT_PASSWORD=secret" \
-    quay.io/minio/minio server /data --console-address ":9001"
-```
-
-Open the Minio console opening in your browser this address `http://localhost:9001`, then create a credentials pair,
-and a bucket. For the sake of this example the bucket name `foobar` will be used.
-
-
-The following configuration should be added in your Nextflow configuration file:
-
-```groovy
-aws.accessKey = '<YOUR MINIO ACCESS KEY>'
-aws.secretKey = '<YOUR MINIO SECRET KEY>'
-aws.client.endpoint = 'http://localhost:9000'
-aws.client.s3PathStyleAccess = true
-wave.enabled = true
+```{code-block} nextflow
+:class: copyable
 fusion.enabled = true
-fusion.exportStorageCredentials = true
-docker.enabled = true
-tower.accessToken = '<your platform access token>' // optional
-```
-
-Then you can run your pipeline using the following command:
-
-```bash
-nextflow run <YOUR PIPELINE> -work-dir s3://foobar/scratch
-```
-
-Replace `<YOUR PIPELINE>` with a pipeline script and bucket of your choice:
-
-:::{warning}
-The option `fusion.exportStorageCredentials` leaks the AWS credentials on the task launcher script created by Nextflow.
-This option should only be used for testing and development purposes.
-:::
-
-### Local execution with Oracle Object Storage
-
-Fusion file system and Nextflow are compatible with [Oracle Object Storage](https://www.oracle.com/cloud/storage/object-storage/).
-
-:::{note}
-This capability relies on the S3-like API compatibility provided by Oracle storage and not by a native support in
-Nextflow and Fusion. As such it may not fully work and support all Nextflow and Fusion features.
-:::
-
-This configuration requires the execution of your pipeline tasks using Docker or a similar container engine.
-
-The following should be included in your Nextflow configuration file:
-
-```groovy
-aws.region = '<YOUR_REGION>'
-aws.accessKey = '<YOUR_ACCESS_KEY>'
-aws.secretKey = '<YOUR_SECRET_KEY>'
-aws.client.endpoint = 'https://<YOUR_BUCKET_NAMESPACE>.compat.objectstorage.<YOUR_REGION>.oraclecloud.com'
-aws.client.s3PathStyleAccess = true
-aws.client.protocol = 'https'
-aws.client.signerOverride = 'AWSS3V4SignerType'
-docker.enabled = true
-docker.containerOptions = '-e FUSION_AWS_REGION=<YOUR_REGION>'
-fusion.enabled = true
-fusion.exportStorageCredentials = true
 wave.enabled = true
-tower.accessToken = '<YOUR_PLATFORM_ACCESS_TOKEN>' // optional
+tower.accessToken = '<PLATFORM_ACCESS_TOKEN>' // Optional
```

Comment:
Platform access token is not optional any more to use Fusion

Reply:
Thanks Paolo, I've removed the note here about it being optional. I'll make sure it's updated in the Fusion docs too.